weave

```diff
@@ -913,21 +913,19 @@ container_weave_addrs() {
 # functions for interacting with containers
 ######################################################################
 
-# Check that a container named $1 with image $2 is not running
+# Check that a container named $1 with image $2 for component $3 is not running
 check_not_running() {
-    case $(docker inspect --format='{{.State.Running}} {{.Config.Image}}' $1 2>/dev/null) in
-        "true $2")
-            echo "$1 is already running." >&2
+    RUN_STATUS=$(docker inspect --format='{{.State.Running}} {{.State.Status}} {{.Config.Image}}' $1 2>/dev/null) || true
+    case ${RUN_STATUS%:*} in
+        "true restarting $2")
+            echo "$1 is restarting; you can stop it with 'weave stop-$3'" >&2
```

Comment:
should have a full stop at end.

Also, perhaps we should print the 'stop' instructions for the non-restarting case too?